core/src/main/resources/application.yml

```diff
@@ -38,7 +38,8 @@ feast:
     runners:
       - name: direct
         type: DirectRunner
-        options: {}
+        options:
+          tempLocation: gs://bucket/tempLocation
```

Comment:
Is this supposed to be under DataflowRunner? Or added here: https://github.com/feast-dev/feast/blob/master/protos/feast/core/Runner.proto#L33

Reply:
it affects Direct as well, and was added here https://github.com/feast-dev/feast/pull/757/files#diff-ed1aa58bf58c59fe211c82b6df89182eR37

Reply:
Oops, I missed that. Thanks for pointing it out.